lombok.config

```diff
@@ -0,0 +1 @@
+lombok.addLombokGeneratedAnnotation = true
```

Comment:
This file excludes any generated Lombok code from the previously implemented Jacoco Test Report

This removes the need to test getter/setter autogenerated code